CONTRIBUTING.md

```diff
@@ -30,16 +30,20 @@ Following that we'll tell you about how you can test your changes locally and th
     It's important to work off the latest changes on the **development** branch.
     ```bash
     # With https
-    git clone https://github.com/your-username/auto-sklearn
+    # Note the --recurse-submodules args, we use a submodule autosklearn/automl_common
+    # so it needs to be downloaded too
+    git clone --recurse-submodules https://github.com/your-username/auto-sklearn
 
     # ... or with ssh
-    git clone git@github.com:your-username/auto-sklearn.git
+    git clone --recurse-submodules git@github.com:your-username/auto-sklearn.git
 
     # Navigate into the cloned repo
     cd auto-sklearn
 
     # Create a new branch based off the development one
     git checkout -b my_new_branch development
+    
+    git submodule udate --init --recursive  # If you missed the --recurse-submodules arg
```

Comment:
What do you think about? This would prevent users from accidentally executing that line (not sure if it would do any harm, though).
```suggestion
    # Execute the following line if you missed the --recurse-submodules arg
    # git submodule udate --init --recursive
```

Reply:
It shouldn't but make's sense